hostpath-provisioner.go

```diff
@@ -84,7 +84,7 @@ func NewHostPathProvisioner(clientset *kubernetes.Clientset) controller.Provisio
 		klog.Fatal("env variable NODE_NAME must be set so that this provisioner can identify itself")
 	}
 
-	pvDir := os.Getenv("PV_DIR")
+	pvDir := os.Getenv("DEFAULT_PV_DIR")
```

Comment:
Let's not change the `PV_DIR` environment variable, backwards compatibility is much preferred here.

Reply:
Ok 👍